src/client/index.ts

```diff
@@ -19,26 +103,82 @@ export function removeEventListener<E extends NotificationEvent>(eventType: E['t
     eventEmitter.removeListener(eventType, listener);
 }
 
+/**
+ * Creates a new Notification.
+ *
+ * The Notification will appear in the Notification Center or as a toast if the Center is not visible.
+ *
+ * ```ts
+ * import {create} from 'openfin-notifications';
+ *
+ * create({
+ *      id: "uniqueNotificationId",
+ *      body: "I'm the Notification body text",
+ *      icon: "https://openfin.co/favicon.ico"
+ * });
+ * ```
+ *
+ * @param options Notification configuration options.
+ */
 export async function create(options: NotificationOptions): Promise<Notification> {
     // Should have some sort of input validation here...
     return tryServiceDispatch(APITopic.CREATE_NOTIFICATION, options);
 }
 
+/**
+ * Clears a specific Notification from the Notification Center.
+ *
+ * ```ts
+ * import {clear} from 'openfin-notifications';
+ *
+ * clear("uniqueNotificationId");
+ * ```
+ *
+ * @param id ID of the Notification to clear.
+ */
 export async function clear(id: string): Promise<boolean> {
     // Should have some sort of input validation here...
     return tryServiceDispatch(APITopic.CLEAR_NOTIFICATION, {id});
 }
 
+/**
+ * Retrieves all Notifications which were generated by your applications uuid.
```

Comment:
I feel like "by your application's uuid" is a weird way to word this. Could we not just say "Retrives all Notifications created by the calling application (including child windows)" or something similar